### PR TITLE
Tickets/DM-31865: JL3

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: jupyterlab/runlab.sh [hunk 1]
```diff
@@ -192,34 +217,37 @@ else
...
 # The Rubin Lap App plus our environment should get the right hub settings
 # This will need to change for JL 3
-cmd="jupyter-rubinlab \
+cmd="jupyter labhub \
      --ip='*' \
      --port=8888 \
      --no-browser \
      --notebook-dir=${HOME} \
-     --LabApp.shutdown_no_activity_timeout=43200 \
+     --hub-prefix='/nb/hub' \
```
Comment (SimonKrughoff) [inline jupyterlab/runlab.sh:225]: I thought frossie wanted this to be just `/nb`

File: jupyterlab/runlab.sh [hunk 2]
```diff
@@ -192,34 +217,37 @@ else
...
      --MappingKernelManager.cull_idle_timeout=43200 \
      --MappingKernelManager.cull_connected=True \
-     --MappingKernelManager.default_kernel_name=lsst \
-     --FileContentsManager.hide_globs=[] \
-     --KernelSpecManager.ensure_native_kernel=False"
-#     --SingleUserNotebookApp.hub_api_url=${EXTERNAL_INSTANCE_URL}${JUPYTERHUB_SERVER_PREFIX}
-#     --SingleUserNotebookApp.hub_prefix=${JUPYTERHUB_SERVICE_PREFIX}
-#     --SingleUserNotebookApp.hub_host=${EXTERNAL_INSTANCE_URL}
+     --MappingKernelManager.default_kernel_name=lsst"
+#     --SingleUserNotebookApp.hub_api_url=${EXTERNAL_INSTANCE_URL}${JUPYTERHUB_SERVER_PREFIX} \
```
Comment (SimonKrughoff) [inline jupyterlab/runlab.sh:233]: Why are we leaving commented out code in?

File: jupyterlab/runlab.sh [hunk 3]
```diff
@@ -192,34 +217,37 @@ else
...
-# echo "JupyterLab command: '${cmd}'"
+echo "JupyterLab command: '${cmd}'"
 # Run idle culler.
-if [ -n "${JUPYTERLAB_IDLE_TIMEOUT}" ] && \
-   [ "${JUPYTERLAB_IDLE_TIMEOUT}" -gt 0 ]; then
-     touch ${HOME}/idleculler/culler.output && \
-       nohup python3 /opt/lsst/software/jupyterlab/selfculler.py >> \
-             ${HOME}/idleculler/culler.output 2>&1 &
-fi
+#if [ -n "${JUPYTERLAB_IDLE_TIMEOUT}" ] && \
```
Comment (SimonKrughoff) [inline jupyterlab/runlab.sh:245]: Same here with the commented out code.

**Review by SimonKrughoff — COMMENTED** [review]
I have a couple of minor questions

File: jupyterlab/runlab.sh (hunk 1 above)
Comment (athornton) [reply jupyterlab/runlab.sh:225]: The _ingress_ is just "/nb".  For this, which is behind the configurable hub proxy, to find the hub, it needs to go to the hub route explicitly.

File: jupyterlab/runlab.sh (hunk 2 above)
Comment (athornton) [reply jupyterlab/runlab.sh:233]: Ah, good point.  I'll remove it.

**Pull request merged**